src/WebJobs.Script.WebHost/Diagnostics/ILinuxAppServiceFileSystem.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Diagnostics
+{
+    public interface ILinuxAppServiceFileSystem
```

Comment:
You're adding this to facilitate mock tests? We already have a pattern for that. You can just use IFileSystem. Look for examples of Mock<IFileSystem> in the tests. I'd rather not have this new type.

Reply:
Fixed.